app/Resources/views/themes/ezplatform/fields/ezrichtext/custom_tags/factbox.html.twig

```diff
@@ -0,0 +1,10 @@
+<div class="factbox">
```

Comment:
AFAIK, there is no design for `factbox`. We will have to provide one.

Reply:
is there something in studio demo perhaps?

Reply:
There is :)
https://github.com/ezsystems/ezplatform-ee-demo/pull/123/files#diff-2c0b103fe4d0dab88bc0bac1938dd24b

Reply:
![image](https://user-images.githubusercontent.com/1157233/55382016-d9a28080-5524-11e9-976b-6f4e2c857517.png)
I am sure Hakan will want to change it. But it can be a starting point

Reply:
Added in ab61a1e